src/main/java/com/epam/ta/reportportal/auth/acl/ShareableObjectsHandler.java

```diff
@@ -61,9 +61,8 @@ public void initAcl(Object object, String owner, Long projectId, boolean isShare
 					.stream()
 					.filter(entry -> !entry.getKey().equalsIgnoreCase(owner))
 					.forEach(entry -> {
-						if (ProjectRole.PROJECT_MANAGER.higherThan(entry.getValue())) {
-							aclService.addPermissions(object, entry.getKey(), BasePermission.READ);
-						} else {
+						aclService.addPermissions(object, entry.getKey(), BasePermission.READ);
+						if (!ProjectRole.PROJECT_MANAGER.higherThan(entry.getValue())) {
```

Comment:
let's make this condition more understandable using for example 'lowerThan'

Reply:
SameOrLower method has been added